tests/unit/source_code/samples/functional/pyspark/dataframe-write-save-as-table.py

```diff
@@ -0,0 +1,13 @@
+# ucx[direct-filesystem-access:+1:0:+1:0] The use of direct filesystem references is deprecated: s3://bucket/path
+df = spark.read.csv("s3://bucket/path")
+for i in range(10):
```

Comment:
i wonder if we want to have a loop in every file. some files might get functions. or methods. etc.

Reply:
Indeed, some variety does no harm.